### ISSUE TITLE
Failed to activate the build-rscript package

### ISSUE THREAD

**Issue by ffdsouza** [idleberg/atom-build-rscript#9]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.54.0 x64
**Electron**: 6.1.12
**OS**: Pop!_OS 20.10
**Thrown From**: [build-rscript](https://github.com/idleberg/atom-build-rscript) package 0.9.0

### Stack Trace

Failed to activate the build-rscript package

```
At Unexpected token ?

/home/ffs76/.atom/packages/build-rscript/lib/provider.js:5
` ut(e,4),super(e),this.name="AggregateError",Object.defineProperty(this,"_errors",{value:t})}*[Symbol.iterator](){for(let t of this._errors)yield t}},ht=U,mt=async(n,t,{concurrency:e=Infinity,stopOnError:r=!0}={})=>new Promise((a,i)=>{if(typeof t!="function")throw new TypeError("Mapper function is required");if(!((Number.isSafeInteger(e)||e===Infinity)

**Comment by idleberg** [comment]
Looks like you need Atom 1.56 or later. I will update the manifest to reflect this requirement.

**Comment by idleberg** [comment]
I've made some changes in `master` that should enable older versions of Atom to consume this package. Could you please confirm it's working for you?

**Comment by idleberg** [comment]
Should be fixed in v0.10.0
